### PR TITLE
#5237 fix added around method to docs

### LINKED ISSUE #5237: [Term Entry] Python NumPy - Math functions .around()
### Reason/inspiration (optional)

We would like a new term entry on the `.around()` function of the [`math-methods`](https://www.codecademy.com/resources/docs/numpy/math-methods) concept in NumPy, Python. The entry should go in a new file under `docs/content/numpy/concepts/math-methods/terms/around/around.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by fazil-shaik**
### Description

I had maken changes on issue and added fix for #5237 a numpy method of around docs is created and motivation behind this documentation is to provide clear, accessible information for learners in computer science and data science, helping them understand and effectively utilize the .around() method in their projects.
### Issue Solved

Closes #5237

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [* ] All writings are my own.
- [* ] My entry follows the Codecademy Docs style guide.
- [ *] My changes generate no new warnings.
- [ *] I have performed a self-review of my own writing and code.
- [ *] I have checked my entry and corrected any misspellings.
- [ *] I have made corresponding changes to the documentation if needed.
- [ *] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ *] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ *] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by fazil-shaik** [comment]
hey hi merge my PR that i had opened assign me this issue

File: content/numpy/concepts/math-methods/terms/around/around.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+| Field | Value |
+| --- | --- |
+| Title | .around() |
+| Description | Rounds each element in an array to the given number of decimals. |
+| Subjects | Computer Science, Data Science |
+| Tags | Arrays, Methods, NumPy, Rounding |
+| CatalogContent | learn-python-3, paths/computer-science |
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/around/around.md:7]:
```suggestion
| Field | Value |
| --- | --- |
| Title | .around() |
| Description | Rounds each element in an array to the given number of decimals. |
| Subjects | Computer Science, Data Science |
| Tags | Arrays, Methods, NumPy, Rounding |
| CatalogContent | learn-python-3, paths/computer-science |
```
Remove all the vertical lines over here. 
 Title: 
Description: 
Subjects:
-
The correct format should be like this.

File: content/numpy/concepts/math-methods/terms/around/around.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Syntax
+
+```pseudo
+numpy.around(x, decimals=0, out=None)
+```
+
+#x: This is the input array containing the numbers you want to round.
+#decimals: The number of decimal places to which each element will be rounded. The default value is 0, which rounds to the nearest integer. Negative values are allowed to round to the left of the decimal point.
+##out: This is an optional parameter. It specifies an output array to store the result. If not provided or set to None, a new array is created for the result.
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/around/around.md:19]:
```suggestion
- x: This is the input array containing the numbers you want to round.
- decimals: The number of decimal places to which each element will be rounded. The default value is 0, which rounds to the nearest integer. Negative values are allowed to round to the left of the decimal point.
- out: This is an optional parameter. It specifies an output array to store the result. If not provided or set to None, a new array is created for the result.
```
Parameters explanation format should be like this.

File: content/numpy/concepts/math-methods/terms/around/around.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+
+Example
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/around/around.md:22]:
```suggestion
## Example
```
Example section should be written using the hash symbol.

File: content/numpy/concepts/math-methods/terms/around/around.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+The output of the above code is shown below:
+
+```
+result_2dec: [1.57 2.35 3.46 4.12]
+result_0dec: [2. 2. 3. 4.]
+result_negdec: [ 0.  0. 0. 0.]
+```
+
+
+Codebyte Example
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/around/around.md:54]:
```suggestion
## Codebyte Example
```

File: content/numpy/concepts/math-methods/terms/around/around.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+import numpy as np
+
+result = np.around(3.14159, decimals=3)
+
+print(result)
+```
+
+In this example, the number 3.14159 is rounded to three decimal places, resulting in 3.142.
+
+```
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/around/around.md:67]:
```suggestion

```

File: content/numpy/concepts/math-methods/terms/around/around.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+This provides a full description of the `.around()` method in NumPy with examples and syntax.
+```
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/around/around.md:69]:
```suggestion

```

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @fazil-shaik ,
I had a look at your file, Please make the suggested changes asap so I can get this PR ready for second round of review.

Thanks,
Savi

**Comment by SaviDahegaonkar** [comment]
Hey @fazil-shaik, Can you please implement the changes I suggested so we can merge this PR?

**Review by SaviDahegaonkar — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @fazil-shaik!

**Pull request merged**